setup.py

```diff
@@ -40,11 +40,11 @@ def get_static_files(path):
 install_requires = [
     'tornado>=4.0,<5',
     'python-daemon<3.0',
+    # So that we can build documentation for luigi.db_task_history and luigi.contrib.sqla
+    'sqlalchemy',
```

Comment:
Interesting this wasn't required. Did we add a reliance on sqlalchemy that didn't exist?

Reply:
When building I had error sqlalchemy import not found. So I figure in our fork something exists that depends on it but we get around because we usually have READTHEDOCS set?

Reply:
Yeah, maybe it's an aspect of the fork. This seems acceptable, though it'd be good to try to pin it if we had a good sense of what version to pin it to.

Reply:
It’s not an aspect of the fork. The main branch uses sqlalchemy to write to the history server. But the Luigi history is optional, so maybe that is why it is not included as a dependency in `setup.py`?